posts/2023-09-26-23.0.0.10-beta.adoc

```diff
@@ -0,0 +1,628 @@
+//
+// However if there's only 1 new feature, delete the previous section and change it to the following sentence:
+// "The link:{url-about}[Open Liberty] 23.0.0.10-beta includes SUB_FEATURE_TITLE"
+// // // // // // // //
+
+See also link:{url-prefix}/blog/?search=beta&key=tag[previous Open Liberty beta blog posts].
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/26059
+// Contact/Reviewer: sebratton,tjwatson
+// // // // // // // // 
+[#sbcrac]
+== Spring Boot 3.0 InstantOn with CRaC beta
+    
+Open Liberty InstantOn provides fast startup times for MicroProfile and Jakarta EE applications. With link:{url-prefix}/blog/2023/06/29/rapid-startup-instanton.html[InstantOn], your applications can start in milliseconds, without compromising on throughput, memory, development-production parity, or Java language features. InstantOn uses the Checkpoint/Restore In Userspace (link:https://criu.org/[CRIU]) feature of the Linux kernel to take a checkpoint of the JVM that can be restored later. 
+
+The Spring Framework (version 6.1) is adding link:https://docs.spring.io/spring-framework/reference/6.1/integration/checkpoint-restore.html[support for Coordinated Restore at Checkpoint (CRaC)] which also uses CRIU in order to provide Checkpoint and Restore for Java applications.  The Spring Boot version 3.2 will use Spring Framework version 6.1, enabling Spring Boot applications to also use CRaC to achieve rapid startup times.
+
+The recent addition of the Open Liberty `springBoot-3.0` feature allows Spring Boot 3.x-based applications to be deployed with Open Liberty.  And now, with the new Open Liberty `crac-1.3` feature, a Spring Boot 3.2-based application can be deployed with Liberty InstantOn in order to achieve rapid startup times for your Spring Boot application.
+
+To use the CRaC 1.3 functionality with the `springBoot-3.0` feature, you must be running with Java 17 or higher and use the `crac-1.3` feature.  Additionally, if your application uses Servlet, it  needs to use the `servlet-6.0` feature.  These features are configured in the `server.xml` file as follows:
+
+[source,xml]
+----
+<features>
+   <feature>springBoot-3.0</feature>
+   <feature>servlet-6.0</feature>
+   <feature>crac-1.3</feature>
+</features>
+----
+
+With these features enabled you can containerize your Spring Boot 3.2 application with Liberty InstantOn support by following the link:{url-prefix}/docs/latest/instanton.html[Liberty InstantOn documentation] along with following the Liberty recommendations for containerizing Spring Boot applications with the link:{url-prefix}/guides/spring-boot.html[Liberty Spring Boot guide].
+
+For more information and an example Spring Boot application using the Liberty InstantOn `crac-1.3` feature see the link:/blog/2023/09/25/spring-boot-3-instant-on.html[How to containerize your Spring Boot application for rapid startup] blog post.

```

Comment:
Need to pay attention to the timing of these blogs.  The general beta blog is set to publish 09/26 and this referenced one is set to publish 9/25.  Are we OK with that timing and we can depend on this link working when published?

Reply:
@dmuelle @GraceJansen FYI the date on the spring-boot-3-instant-on post.